src/components/category.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react"
+import styled from "styled-components"
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faCircle } from '@fortawesome/free-solid-svg-icons'
+
+export const CategoryWrapper = styled.div`
```

Comment:
This component is in src/components/card.tsx, maybe you can centralize all common components that are used in different components in one place

Reply:
I just noticed the one in src/components/card.tsx wasn't necessary, hehe. Thanks for the suggestion though, and good catch 😅